sandbox/Devis/Keyvault.tf

```
@@ -0,0 +1,32 @@
+resource "azurerm_key_vault" "kv_azure_demo_01" {
```

No need to include resource type in terraform resource names, as type is already there in first part
```suggestion
resource "azurerm_key_vault" "azure_demo_01" {
```